Civi/Funding/PayoutProcess/Validator/DrawdownValidator.php

```diff
@@ -63,6 +67,27 @@ public function __construct(FundingCaseManager $fundingCaseManager, PayoutProces
    * phpcs:disable Drupal.Commenting.FunctionComment.IncorrectTypeHint
    */
   public function validate(AbstractEntity $new, AbstractEntity $current): EntityValidationResult {
+    $payoutProcess = $this->getPayoutProcess($new);
+    $this->assertNotClosed($payoutProcess);
+
+    if ($new->getAmount() < 0) {
+      $payoutProcess = $this->getPayoutProcess($new);
+      $fundingCase = $this->getFundingCase($payoutProcess);
+
+      if (!$this->hasReviewPermission($fundingCase->getPermissions())) {
+        return $this->createAmountLessThanZeroResult();
+      }
+    }
+
+    if ($new->getAmount() > $current->getAmount()) {
+      $payoutProcess = $this->payoutProcessManager->get($new->getPayoutProcessId());
+      Assert::notNull($payoutProcess);
+      $amountDiff = $new->getAmount() - $current->getAmount();
```

Comment:
is it necessary to handle negative values in both amounts?

Reply:
I'm not sure what you mean.

Reply:
can it be that both `getAmount()` calls return negative values?
ie: new=-2, current=-3
=> new > current
=> -2 - -3 = 1